gemini-chat-app.py

```diff
@@ -1,15 +1,27 @@
 #TODO: Install before running
 #pip install gradio
 #pip install google.generativeai
+#pip install gradio_pdf
 
 #run code - python gemini-chat-app.py
 
 import gradio as gr
 import os
 import google.generativeai as genai
 
-def gemini_chat(message, history):
-    response = chat.send_message(message)
+
+#custom theme
+theme = gr.themes.Base(
+    secondary_hue="gray",
+).set(
+    border_color_accent_dark='*neutral_800'
+)
```

Comment:
We can customize the theme to the one used by Judith 

To create the theme we can use - [](https://www.gradio.app/guides/theming-guide)